### PR TITLE
pointcloud_octomap_updater: Don't return false when not finding optional parameter

### PR DESCRIPTION
**Pull request by JafarAbdi**
### Description

It was causing the function to return false even if we are missing an optinal parameter

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: moveit/package.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>moveit</name>
-  <version>2.5.1</version>
```
Comment (tylerjw) [inline moveit/package.xml:5]: ?

**Review by tylerjw — CHANGES_REQUESTED** [review]
I think you must have accidentally included a bunch of version bumps in this?

**Review by MarqRazz — APPROVED** [review]
This fixes errors on my system running Humble.

File: moveit_ros/perception/pointcloud_octomap_updater/src/pointcloud_octomap_updater.cpp [hunk 2]
```diff
@@ -72,14 +72,14 @@ PointCloudOctomapUpdater::~PointCloudOctomapUpdater()
 
 bool PointCloudOctomapUpdater::setParams(const std::string& name_space)
 {
+  node_->get_parameter_or(name_space + ".ns", ns_, std::string());
```
Comment (AndyZe) [inline moveit_ros/perception/pointcloud_octomap_updater/src/pointcloud_octomap_updater.cpp:75]: Please add a comment here so we can follow this logic in the future.

```suggestion
  // This parameter is optional
  node_->get_parameter_or(name_space + ".ns", ns_, std::string());
```

**Review by AndyZe — APPROVED** [review]

**Review by vatanaksoytezer — APPROVED** [review]

**Review by tylerjw — APPROVED** [review]

**Pull request merged**